install/aiven/04_configure_the_collector_package.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Step 4: Configure the Collector'
+install_track_title: Installation Guide (Aiven)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+import imgAivenDbSettings from './aiven-db-settings.png'
+
+export const ImgAivenDbSettings = () => <img alt="Aiven database settings screenshot" src={imgAivenDbSettings} />
+
+import PublicLastStepLogInsightsLink from '../_public_last_step_log_insights_link.mdx'
+
+export const APIKeyInstructions = ({ apiKey }) => {
+  return apiKey ? (
+    <React.Fragment>
+      The <code>api_key</code> for the current organization is{" "}
+      <code>{apiKey}</code>
+    </React.Fragment>
+  ) : (
+    <React.Fragment>
+      The <code>api_key</code> can be found in the pganalyze Settings page for
+      your organization, under the API keys tab
+    </React.Fragment>
+  );
+};
+
+export const ConfigSample = ({ apiKey }) => {
+  return (
+    <CodeBlock>
+      {`[pganalyze]
+api_key = ${apiKey ?? 'your_pga_organization_api_key'}\n
+[server1]
+db_host = your_database_host
+db_port = your_database_port
+db_name = your_database_name
+db_username = your_monitoring_user
```

Comment:
Yeah, we could standardize on 'pganalyze' in examples, but I have seen support interactions with monitoring users with other names, so that's not necessarily a clear win. In either case, I think we should stay consistent with other instructions for now and address this separately.